src/Symfony/Component/BrowserKit/AbstractBrowser.php

```diff
@@ -161,6 +161,24 @@ public function xmlHttpRequest(string $method, string $uri, array $parameters =
         }
     }
 
+    /**
+     * Converts the request parameters into a json string and use it as request content.
+     */
+    public function jsonRequest(string $method, string $uri, array $parameters = [], array $files = [], array $server = [], bool $changeHistory = true): Crawler
```

Comment:
hmmm. I'm not sure we can upload `files` with an `application/json` content-type.
I would remove this parameter.

Reply:
Sounds reasonable did change it.